components/question/Textarea/index.jsx

```diff
@@ -0,0 +1,45 @@
+import PropTypes from 'prop-types';
+import { useEffect, useState } from 'react';
+
+import { buildSurveyQuestionsWithAnswers } from '../Details/buildSurveyQuestionsWithAnswers';
+
+export const textareaTestIds = {
+  base: 'textarea',
+  inputField: 'textarea__input-field',
+};
+
+const Textarea = ({ id, answers, setSurveyQuestionsWithAnswers }) => {
+  const [inputValue, setInputValue] = useState('');
+
+  useEffect(() => {
+    setInputValue('');
+  }, [id]);
+
+  const onInputValueChange = value => {
+    const surveyQuestionsWithAnswers = buildSurveyQuestionsWithAnswers(id, answers[0].id, value);
+    setInputValue(value);
+    setSurveyQuestionsWithAnswers(surveyQuestionWithAnswer => {
+      surveyQuestionWithAnswer = surveyQuestionWithAnswer.filter(response => response.id !== id);
+      return [...surveyQuestionWithAnswer, surveyQuestionsWithAnswers];
+    });
+  };
+
+  return (
+    <div className="h-[112px]" data-test-id={textareaTestIds.base}>
+      <textarea
+        className="w-full h-full bg-dark text-white text-base-large rounded-xl px-3 pt-[19px] pb-[15px] mt-16 focus:outline-none focus:shadow-none"
```

Comment:
Can we move the `height` to the `textarea` directly? Not so sure why do we need to wrap the `textarea` with a `div` 😅

Reply:
sure, done in 159656fc3e35af63bc6b4f46934c77839918a438, Thanks!